src/App.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import { useSelector, useDispatch } from 'react-redux';
+
+import { render } from '@testing-library/react';
+
+import App from './App';
+
+describe('App', () => {
+  const dispatch = jest.fn();
+
+  beforeEach(() => { jest.clearAllMocks(); });
```

Comment:
줄바꿈 처리를 하거나 한줄로 사용하려면 아래와 같이해줘도 되지 않을까요?
```suggestion
  beforeEach(() => jest.clearAllMocks() );
```